features/google/src/views/report/reportWrapper.jsx

```diff
@@ -23,10 +25,11 @@ const ReportWrapper = () => {
     const activeReportStories = computedReportStoriesList.filter(
         (reportStory) => reportStory.active
     );
-    const statusReport = new MinistoriesStatusReport([
-        ...computedReportStoriesList,
-        ...computedMinistories,
-    ]);
+    const statusReport = new MinistoriesStatusReport({
+        ministories: [...computedReportStoriesList, ...computedMinistories],
```

Comment:
This is the other call but, again, can't see how this is wrong. Some typo I'm missing?